### PR TITLE
Bug/saml proxy handle err

### PR DESCRIPTION
**Pull request by crolarlibertyva**
https://vajira.max.gov/browse/API-950

### CODE REVIEW

**Comment by rtravitz** [comment]
If it's helpful, we have an example of an express default error handler we've been trying to improve in the developer-portal-backend. https://github.com/department-of-veterans-affairs/developer-portal-backend/blob/a161f029a96fe55122ed770630f90af3af5cd2d4/app.ts#L155

And the Sentry Express middleware: https://github.com/department-of-veterans-affairs/developer-portal-backend/blob/a161f029a96fe55122ed770630f90af3af5cd2d4/app.ts#L147

File: saml-proxy/docker-compose.yml [hunk 1]
```diff
@@ -6,5 +6,4 @@ services:
       dockerfile: saml-proxy/Dockerfile
     ports:
       - "7000:7000"
-    network_mode: "host"
```
Comment (lukemarkwordtlibertyits) [inline saml-proxy/docker-compose.yml:9]: Why remove network_mode?

File: saml-proxy/views/handleFailure.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {{#>deptva-formation-error}}
   <h3 class="usa-alert-heading">
-    Some VA.gov tools aren't working right now
+    Some VA.gov service applications aren't working right now. Please note
```
Comment (beaugrantham-va) [inline saml-proxy/views/handleFailure.hbs:5]: Not sure why this was changed... doesn't seem to add any value.

File: saml-proxy/docker-compose.yml (hunk 1 above)
Comment (crolarlibertyva) [reply saml-proxy/docker-compose.yml:9]: When running in the host network mode the docker instance is not reachable via "localhost:7000" when running on docker for desktop in windows. Plus this made it consistent with oauth_proxy.

File: saml-proxy/views/handleFailure.hbs (hunk 2 above)
Comment (crolarlibertyva) [reply saml-proxy/views/handleFailure.hbs:5]: Will revert.

**Pull request merged**